current/README.md

```diff
@@ -12,7 +12,11 @@ Our documentation is divided into several main sections, which cover the Archive
 
 ### Getting Started
 
-- [Installing the Archives Unleashed Toolkit](https://github.com/archivesunleashed/aut-docs-new/blob/master/current/install.md)
+- [Setting up the Archives Unleashed Toolkit](setting-up-aut.md)
+- [Using the Archives Unleashed Toolkit at Scale](aut-at-scale.md)
+- [Archives Unleashed Toolkit Lesson](toolkit-lesson.md)
+- [Docker Installation](docker-install.md)
+- [Cookbook](cookbook.md)
```

Comment:
Our audience might not know what a cookbook is - should we add a few words in parentheses?

Reply:
I like having it somewhere on the ToC - what about like Cookbook (short scripts for reference) or something?

Reply:
Yep that’s what I had in mind - should have been clearer!